tools/webdriver/webdriver/bidi/modules/script.py

```diff
@@ -0,0 +1,53 @@
+from typing import Any, Optional, Mapping, MutableMapping, Union, Dict
+from enum import Enum
+
+from ._module import BidiModule, command
+
+
+class ScriptResultException(Exception):
```

Comment:
nit: probably rename to ScriptEvaluateResultException to match https://github.com/w3c/webdriver-bidi/pull/224

Reply:
done

Reply:
done